### PR TITLE
Update openSUSE packaging and building info

### PR DESCRIPTION
**Pull request by tujhen**
Finally I have time to clean up docs about openSUSE.

Changed openSSL dependency to mainstream, added info about new deps for build, added some info with notes about issues for package installation. Also I package python-hwdata and provide link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,7 +110,9 @@ You can use the `tuxclocker.tar` from the release page if you don't want to comp
 This list may contain unofficial distribution packages, maintained by other people not directly related with this project.
 
 #### For openSUSE Tumbleweed
-Install the [tuxclocker](https://software.opensuse.org//download.html?project=home%3Atujhen%3Atuxclocker&package=tuxclocker) package from OBS. Or via terminal
+NOTE: Due to NVIDIA proprietary driver license problems package builded without NVIDIA modules. See [issue#65](https://github.com/Lurkki14/tuxclocker/issues/65) for more info.
```
Comment (Lurkki14) [inline README.md:113]:
```suggestion
NOTE: Due to NVIDIA proprietary driver license problems the package is built without NVIDIA modules. See [issue#65](https://github.com/Lurkki14/tuxclocker/issues/65) for more info.
```

File: README.md [hunk 2]
```diff
@@ -172,9 +176,10 @@ sudo zypper in -y \
     libQt5DBus-devel \
     libQt5Charts5-devel \
     libqt5-qtbase-devel \
-    libopenssl-1_0_0-devel \
-    python3-devel \
-    python-hwdata
+    libopenssl-devel \
+    python311-devel \
+    python311-hwdata \
```
Comment (Lurkki14) [inline README.md:181]: Why `python311` and not `python3` ?

File: README.md [hunk 3]
```diff
@@ -159,9 +161,11 @@ sudo apt install --yes --quiet --quiet \
 ```
 
 #### For openSUSE Tumbleweed
+`python-hwdata` is not packaged for openSUSE but can be acquired on [OBS](https://build.opensuse.org/package/show/home:tujhen:tuxclocker/python-hwdata).
```
Comment (Lurkki14) [inline README.md:164]:
```suggestion
`python-hwdata` 2.4.1 is not packaged for openSUSE but can be acquired on [OBS](https://build.opensuse.org/package/show/home:tujhen:tuxclocker/python-hwdata).
```
It is packaged from what I can see, but 2.4.1 is preferred

**Review by Lurkki14 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (tujhen) [reply README.md:181]: openSUSE has own packaging rules for Python and building process. We don't have something like python3-devel, we have package python-devel, that provides 3 packages (for python 3.9, 3.10 and 3.11). Current main Python in openSUE Tumbleweed is 3.11 

Link for packaging rules:
https://en.opensuse.org/openSUSE:Packaging_Python#What_is_single-spec?

File: README.md (hunk 3 above)
Comment (tujhen) [reply README.md:164]: Don't understand in some moments, but as I explain upper, I package python-hwdata package with latest version that provides 3 packages for python 3.9, 3.10, 3.11

File: README.md (hunk 2 above)
Comment (tujhen) [reply README.md:181]: In fact of that, we have 4 different pythons with modules in repo at that point, that marked by version (python39-notify2 for example, not python3)

File: README.md (hunk 3 above)
Comment (tujhen) [reply README.md:164]: Not in official (main repo), can't find with zypper, only by opi older in semi-main and latest in home:repos (OBS)

https://software.opensuse.org/package/python-hwdata?search_term=python-hwdata

File: README.md (hunk 1 above)
Comment (Lurkki14) [reply README.md:113]: Other stuff looks good to merge, this needs changing still

Comment (tujhen) [reply README.md]: Seems done

**Comment by Lurkki14** [comment]
Squash the commits into one, please

**Comment by tujhen** [comment]
Can you squash it manually? I have a little trouble to squash it from web, seems you need to close "Changes requested" and after "Squash and merge"

**Comment by Lurkki14** [comment]
That works too, wasn't aware of this button

**Comment by pallaswept** [comment]
Nice work mate. I've pulled my build, I don't want to get a ban from OBS for having nvidia there.

Just a heads-up - you forgot the desktop icon which @Lurkki14 added just recently. They document it all quite nicely here: https://en.opensuse.org/openSUSE:Packaging_guidelines#Desktop_files (you have to follow three links to see it all, but it starts there) it's easy for a guy like you :)

**Pull request merged**